contents/blog/hiring-myths-and-truths.md

```diff
@@ -0,0 +1,75 @@
+
+## Truth #1: The bar for writing good job ads is depressingly low
+
+Follow this devastatingly simple process:
+
+- Outline the _exact_ process they will follow (meet X person by name, do Y task etc.). Do not make the process up for each candidate. 
+- Write out what they will be working on in specific terms - don't just list generic responsibilities. We find linking to recent GitHub Issues a fantastic way of doing this. 
+- Include a salary range. If you can't afford to pay a good salary, then let expensive people self-select out - it's ok!
+
+Congrats - you are now beating 90% of specs out there. Optional extra: add some personality/weirdness. 
+
+Don't just let your recruiter write the spec without help - they are not copywriters with expertise in every technical discipline ever. Treat it with more importance and care as marketing material - recruiting is more leveraged than blog posts, and yet people put in half the effort into the ad copy. 
+
+You want people to filter themselves out of the process if the work isn't appealing to them. This will save you time in the long run. 
+
+## Truth #2: getting people to do actual (paid) work for you is 10x more effective than the rest of your interviews combined
+
+Interviews are imperfect in _so_ many ways, and should be generally be seen as a reasonably effective filtering mechanism. Getting candidates to do a sustained piece of actual work in a semi-realistic context is much better. Pay them for their time, obviously - it's cheaper than recruiter fees. 
+
+This also levels the playing field, because a lot of interviews only tell you if this person is good at giving interviews. It's also a great way to sniff out exec-speak BS when you are hiring VP of whatever from fancy massive company - you'll very quickly find out if they can do the work. Don't just make the work 'come up with a strategy' - ask people to actually do the work.
+
+Interviews still provide useful information - you just shouldn't prioritize them at the decision-making stage.
+
+## Truth #3: Most recruitment metrics are vanity metrics
+
+Number of applications, number of initial phone screens, number of candidates sourced - these are all vanity metrics. It is extremely easy to get 500+ low quality applications for a role - even specialist ones. In marketing, it's the equivalent for optimizing for pageviews and ignoring conversion to paid. Don't do it. 
+
+I once managed to get 900 applications in 48 hours for an ops role at a company with pretty low brand recognition that was only advertising on LinkedIn with minimal budget. That doesn't make me a good recruiter. 
+
+The number of people at late/final stages of your process, and their conversion rates to offers made are the only numbers that really matter.
```

Comment:
In the absence of a conclusion, I like to add a 'further reading' section of recommended reading on the topic. Should include a mix of our own stuff (if relevant) and good reads from other sources.